techbooksdotd.rb

```diff
@@ -79,13 +79,28 @@
 
 def get_deals
   deals = Array.new
-  deals << get_apress(open('http://www.apress.com/').read)
+  
+  begin
+	apressContent = open('http://www.apress.com/').read
+	apressContent = get_apress(apressContent)
+  rescue Exception => e
+	apressContent = $apress_deal
+  end
+  deals << apressContent
```

Comment:
I haven't done Ruby in so long, but won't `apressContent` be out of scope here?

Reply:
The script still works, but point taken. I've updated the script so the variables are initialized to empty strings.

Reply:
@hoop33 PR has been updated.